src/main/target/TINYFISH/target.h

```diff
@@ -66,7 +66,7 @@
 #define UART3_TX_PIN            PB10
 #define UART3_RX_PIN            PB11
 
-#define SBUS_TELEMETRY_UART     SERIAL_PORT_USART2
+#define SBUS_AND_TELEMETRY_UART SERIAL_PORT_USART2
```

Comment:
ok.

this still leaves the codebase in an inconsistent state with ragets to SERIALRX_UART however.  That define causes a similar problem but would require ADDING config.c files for pretty much every target.